stories/module-optimize-setup.stories.js

```diff
@@ -61,40 +44,46 @@ function Setup( props ) {
 }
 
 storiesOf( 'Optimize Module/Setup', module )
-	.add( 'Start', () => {
-		filterOptimizeSetup();
+	.addDecorator( ( storyFn ) => {
+		const registry = createTestRegistry();
+		global._googlesitekitLegacyData.setup.moduleToSetup = 'optimize';
+		registry.dispatch( CORE_MODULES ).receiveGetModules( [
+			{
+				slug: 'optimize',
+				active: true,
+				connected: true,
+			},
+		] );
+		registry.dispatch( CORE_MODULES ).registerModule( 'optimize', {
+			setupComponent: OptimizeSetup,
+		} );
 
-		const setupRegistry = ( { dispatch } ) => {
-			dispatch( CORE_MODULES ).receiveGetModules( analyticsFixture );
-			dispatch( MODULES_ANALYTICS ).setUseSnippet( true );
-			dispatch( STORE_NAME ).receiveGetSettings( {} );
-		};
+		return storyFn( registry );
+	} )
+	.add( 'Start', ( registry ) => {
+		registry.dispatch( CORE_MODULES ).receiveGetModules( analyticsFixture );
```

Comment:
This is somewhat incorrect (or at least confusing) because it will set the modules so that only Analytics exists and is active, overriding the data provided in the decorator that makes Optimize the only available and active module. Since Optimize requires Analytics to be active, we should probably remove all these clauses and instead in the decorator use `provideModules` with both `analytics` _and_ `optimize`.